src/authz-module/data/hooks.ts

```diff
@@ -40,3 +45,42 @@ export const useLibrary = (libraryId: string) => useSuspenseQuery<LibraryMetadat
   queryFn: () => getLibrary(libraryId),
   retry: false,
 });
+
+/**
+ * React Query hook to add new team members to a specific library.
```

Comment:
Same comment for both hooks https://github.com/openedx/frontend-app-admin-console/pull/3/files#r2389701104